tests/test_surface_sphere_project_unproject.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import os
+from pathlib import Path
+
+from styxdefs import set_global_runner
+from styxsingularity import SingularityRunner
+from niwrap import workbench as wb
+import nibabel as nib
+
+from neuromaps_nhp.utils.surface_sphere_project_unproject import surface_sphere_project_unproject
+from neuromaps_nhp.config import config
+
+@pytest.mark.parametrize(
```

Comment:
This isn't quite how we want to use parametrize unless you are adding additional cases so that we can test multiple cases.